src/components/Resource/ResourceBoard.tsx

```diff
@@ -92,10 +93,14 @@ export default function BoardView() {
             currentTab={boardFilter !== ACTIVE ? 1 : 0}
           />
           <div className="flex w-full flex-col gap-2 lg:mr-4 lg:w-fit lg:flex-row lg:gap-4">
-            <ButtonV2 className="py-[11px]" onClick={onListViewBtnClick}>
+            <Button
+              className="h-[45px]"
+              variant={"primary"}
+              onClick={onListViewBtnClick}
+            >
```

Comment:
let's stick with button's default size. 

```suggestion
            <Button
              variant={"primary"}
              onClick={onListViewBtnClick}
            >
```

Reply:
Okay, will update it